src/codeCache.h

```diff
@@ -103,7 +109,7 @@ class CodeCache {
     unsigned int _plt_offset;
     unsigned int _plt_size;
 
-    void** _imports[NUM_IMPORTS];
+    void** _imports[NUM_IMPORT_TYPES * NUM_IMPORTS];
```

Comment:
Maybe `_imports[NUM_IMPORT_TYPES][NUM_IMPORTS]`? for better readability and to avoid manual index calculation.